docs/migration/manual-migration.mdx

```diff
@@ -23,9 +23,27 @@ If you use SEO components or have overriden Seo components, you need to:
 - Remove import and usage of the component from your page
 - Use the [meta](https://remix.run/docs/en/main/route/meta-v2)
   function with our
-  [generateMetas helper](/docs/remixed/api-reference/front-commerce-remix/generate-metas) 
+  [generateMetas helper](/docs/remixed/api-reference/front-commerce-remix/generate-metas)
   to inject your custom metadatas in your Route Module.
 
+### `Enhancers` components
```

Comment:
**nitpick:** similar to SEO components, it's also related to the more global shift to using Remix routes.

**suggestion(for-now):** rename the "Components" section to a "Refactor pages to Remix routes"

**suggestion(long-term):** it'll very likely be its own page!